mlptrain/sampling/md.py

```diff
@@ -339,14 +355,32 @@ def _run_dynamics(
     n_steps: int,
     energies: List,
     biased_energies: List,
+    pressure: Optional[float] = None,
+    compress: Optional[float] = None,
     **kwargs,
 ) -> None:
     """Initialise dynamics object and run dynamics"""
 
+    if all([value is not None for value in [pressure, compress]]) and temp > 0:
```

Comment:
I believe this should be connected to the next condition, e.g., the one resolving NVE vs NpT

Reply:
Indeed, I will change the other if to an else